src/Microsoft.WSMan.Management/WsManHelper.cs

```diff
@@ -851,10 +851,10 @@ internal void CleanUp()
             }
         }
 
-        internal string GetFilterString(Hashtable seletorset)
+        internal string GetFilterString(Hashtable selectorset)
         {
             StringBuilder filter = new StringBuilder();
-            foreach (DictionaryEntry entry in seletorset)
+            foreach (DictionaryEntry entry insselectorset)
```

Comment:
This doesn't look right

It would be a lot easier to review if you split this into two PRs.  One for comments and one for code.  The comments should be easier to review.